### DISCUSSION TITLE
RxJs 7

### DISCUSSION THREAD

**Discussion by jan-kacina** [marblejs/marble#330] [Q&A] [answered]
Hi guys,
Do you plan to migrate to RxJs 7? If yes, when we can expect it?

Thanks and regards,
Jan

**Comment by JozefFlakus** [comment, marked as answer]
Hi!
The work is already started. I've just did it internally and took me around 2 hours to add the initial support (without some deprecation changes). As discussed here #324 - we have to introduce it with next major version release. I will keep you informed about the progress.

**Reply by JozefFlakus** [reply]
CC #332

**Reply by JozefFlakus** [reply]
Marking it as `answered` because `RxJS v7` support is available in `v4.0` release candidate.
